### PR TITLE
[18Scan] Add initial game files

### PR DESCRIPTION
**Pull request by vandamm**
This PR adds initial files for 18Scan. 

Map and entities are done, more work to follow on title specific items: different auction type, private abilities, destination runs and SJ merger.

I'm doing this in parts to make review of the code easier.

### CODE REVIEW

File: lib/engine/game/g_18_scan/step/buy_sell_par_shares.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  module Game
+    module G18Scan
+      module Step
+        class BuySellParShares < Engine::Step::BuySellParShares
+          SJ_START_PRICE = 100
+
+          def get_par_prices(entity, corp)
+            return super unless corp == @game.sj
+
+            @game.stock_market.par_prices.filter do |p|
```
Comment (tobymao) [inline lib/engine/game/g_18_scan/step/buy_sell_par_shares.rb:15]: what does filter do, use either reject or select

File: public/logos/18_scan/1.alt.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 8 8">
```
Comment (tobymao) [inline public/logos/18_scan/1.alt.svg:1]: minify your svgs

File: lib/engine/game/g_18_scan/step/buy_sell_par_shares.rb (hunk 1 above)
Comment (vandamm) [reply lib/engine/game/g_18_scan/step/buy_sell_par_shares.rb]: done

File: public/logos/18_scan/1.alt.svg (hunk 2 above)
Comment (vandamm) [reply public/logos/18_scan/1.alt.svg]: done

**Pull request merged**